lib/fetch/index.js

```diff
@@ -92,13 +92,13 @@ export function fetchTalk(args) {
 }
 
 // TODO: Test? lol
```

Comment:
We should not get rid of this comment without fixing this issue.

We should fix this, but not in this unrelated PR.